app/views/templates/content.jst

```diff
@@ -0,0 +1,2 @@
+<div role="header"></div>
```

Comment:
Maybe use [`role=complementary`](https://www.w3.org/TR/wai-aria/roles#complementary) to dipslay additonal informations (role `header` doesn't exists).